### ISSUE TITLE
Application setup instructions

### ISSUE THREAD

**Issue by olivierobert** [Quentin-34/MyGoogleParser#18]
Note that I failed to install the application on my local development machine due to the following:
- While I use a Mac, I only successfully ran .Net Core application on version 5.x and above. The following error is displayed which seems to suggest that the application requires an older version:

![image](https://user-images.githubusercontent.com/696529/140703553-4ee77164-1fa6-4142-a435-dce0b9c09d40.png)

If that's the case, why use an older version since 5.x is now the recommended version?

![image](https://user-images.githubusercontent.com/696529/140703182-a8daa0ce-7189-49db-9823-adf7a8a2bf0b.png)

The README file only mentioned .NET Core SDK but actually, a specific version seems required

- The database instructions are incomplete since [the SQL script](https://github.com/Quentin-34/MyGoogleParser/blob/main/PostgreSQL/CreateTable.pgsql) only create a single table. The database creation portions is missing. 

Instead of using a local database, have you considered using Docker and Docker Compose to easily set up dependencies lie databases? Do you have experience in using these tools?

Overall, more information in the README would be a significant improvement for developers other than you - the author.

**Comment by Quentin-34** [comment]
You're right, .NET 5.0 would have been more appropriate. I had in mind that .NET Core 3.1 was the Long Term Support version with the longest support but I didn't check about the latest recommended versions

**Comment by olivierobert** [comment]
Noted. I still did not manage to make it run locally 😅  I gave up on .NET Core 3.1 for my Mac. Based on my Google search, it would require removing entirely .Net Core application on version 5.x  and then a complete new installation for 3.1 😵‍💫

**Comment by Quentin-34** [comment]
Sorry that you cannot run the app.
By running `npm install`, then `npm run serve`. You should be able to access to VueJs app only: http://localhost:8080/
It won't be functional but at least you can have look
Should I post here screenshots of other pages?

**Comment by olivierobert** [comment]
I did manage to run the client application:

![image](https://user-images.githubusercontent.com/696529/140871641-cdf8662a-d4f1-4d33-b10e-eedc4fda8e03.png)

However, I cannot go very far since the sign in with Auth0 leads to an error:

![image](https://user-images.githubusercontent.com/696529/140871732-5c8e9828-098f-43e3-ae50-ed51137372b2.png)

So if you can add some images of the upload screen and list of keywords that would be good 👍

**Comment by Quentin-34** [comment]
Once logged in, user menu and navigation links are displayed

![image](https://user-images.githubusercontent.com/93501831/140874914-ee767e70-f70a-46ae-8e64-9aaf5ae23e6c.png)

**Comment by Quentin-34** [comment]
Keywords list, with instant search on any fields

![image](https://user-images.githubusercontent.com/93501831/140875434-25480a86-dbc0-4273-a7ef-b43a88f89314.png)

**Comment by Quentin-34** [comment]
Upload page

![image](https://user-images.githubusercontent.com/93501831/140875540-5e42dce1-5bd4-4701-8760-28688de4cbf9.png)
